app/javascript/components/recipes/base/recipe-ingredients.vue

```diff
@@ -12,7 +12,6 @@
         >
         <!-- ingredientes disponibles -->
         <div class="flex flex-col items-start w-auto h-96 flex-none flex-grow-0 bg-gray-200 overflow-scroll">
-        <!-- <div class="flex flex-col bg-gray-200 overflow-scroll"> -->
```

Comment:
Ahí caché que la borrase. Para que no pasen esas cosas es mejor hacer un git rebase con el arreglo. Básicamente haces un `git add`  del arreglo al commit, después `git comit --fixup=hash-del-commit-que-quieres-arreglar` y de ahí  `git rebase -i --autosquash hash-del-commit-justo-antes-del-que-quieres-arreglar`. Si el commit que arreglaste ya está pusheado, haces después un `git push --force-with-lease origin nombre-de-tu-branch`. En este post Raimundo lo explica mucho mejor : https://blog.platan.us/manteniendo-la-historia-limpia-usando-git-rebase-5cceecd5367a?gi=7764c727a95c

Reply:
Para arreglar la historia en este caso, como son solo 2 commits quizás es más fácil volver a hacerlos nomás:
- `git reset HEAD ~2`
- `git add` sólo con los cambios de recipe-ingredients,
- `git commit -m 'fix(recipes): change height of ingredients and add scrollbar'`
- `git add `sólo con los cambios de menu-container
- `git commit -m 'fix(menus): change size of search input'
- Los cambios del schema los agregaría en un commit aparte con el prefijo "style", ya que no cambian nada